extensions/not-diamond/CHANGELOG.md

```diff
@@ -0,0 +1,3 @@
+# Not Diamond Changelog
+
+## [Initial Version] - {PR_MERGE_DATE}
```

Comment:
Please set the current data if you can't change the modify rights for our bot 🙂

Reply:
This is done ✅